Reyes_JohnRuiz/index.html

```diff
@@ -0,0 +1,184 @@
+          <li><a href="./work/Laws.html">Work 4</a></li>
+          <li><a href="./work/marketing.html">Work 5</a></li>
+        </ol>
+      </section>
+
+      <section id="testimonials">
+        <h2>Testimonials</h2>
+        <div class="testimonials">
+          <div>
+            <p>
+              Bergado, Shamma S.J. Monday, April 8, 2024 Working alongside my
+              coding group mate Ruiz has been nothing short of inspiring. His
+              passion for problem-solving and dedication to excellence have made
+              every coding session a rewarding experience.
+            </p>
+          </div>
+
+          <div>
+            <p>
+              Norona, Mary Nicole P. Monday, April 8, 2024 Having a teammate
+              like John, who is a very responsible and hardworking man is one of
+              the reasons why we are creating great projects in our team.
+            </p>
+          </div>
+
+          <div>
+            <p>
+              Reyes, Paul Angelo Monday, April 8, 2024 Coding with my group
+              mates has been smooth sailing. Their exceptional skills and
+              seamless communication help us tackle challenges with confidence
+              and achieve remarkable results.
+            </p>
+          </div>
+
+          <div>
+            <p>
+              Morcoso, Daisy Monday, April 8, 2024 Ruiz, my classmate, has
+              potential in coding. While not advanced, he grasps programming
+              concepts and shows promise in class projects. With further
+              growth,he could become a valuable asset to our coding team.
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="gallery">
+        <h2>Gallery</h2>
+        <div class="images">
+          <img src="Joji_-_Nectar.png" alt="Image 1" />
+          <img src="billie.jpg" alt="Image 2" />
+          <img src="gameofthrones.jpg" alt="Image 3" />
+          <img src="skyline.jpg" alt="Image 4" />
+          <img src="zild.jpg" alt="Image 5" />
+          <img src="strnger.jpg" alt="Image 6" />
+        </div>
+      </section>
+
+      <section class="my-goals">
+        <h2>My Goals</h2>
+        <ol>
+          <li>To be a Developer</li>
+          <li>To learn some new Skills</li>
+          <li>to travel outside of country </li>
+          <li>to </li>
+          <ol>To be happy</ol>
+        </ol>
+      </section>
+
+      <section>
+        <h2>Add a comment here</h2>
+        <form id="comment-form" class="add-comment">
+          <input
+            type="text"
+            name="name2"
+            id="comment-name"
+            placeholder="Enter your name"
+          />
+          <textarea
+            name="textarea-for-comment"
+            id="comment-text"
+            placeholder="Add your comment here"
+          >
+          </textarea>
+          <button type="button" name="comment" id="submit-comment">Comment</button>
+        </form>
+      </section>
```

Comment:
IDs should be written in snake case. All lower case separated by underscore. Remove excess line spacing as much as possible. See the reference and suggested code for clarifications.
```suggestion
            name="name2"
            id="comment_name"
            placeholder="Enter your name"/>
          <textarea
            name="textarea-for-comment"
            id="comment_text"
            placeholder="Add your comment here">
          </textarea>
          <button type="button" name="comment" id="submit_comment">Comment</button>
        </form>
      </section>
```

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5